erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py

```diff
@@ -19,8 +19,6 @@ def _execute(filters=None, additional_table_columns=None, additional_query_colum
 	company_currency = frappe.get_cached_value('Company',  filters.get("company"),  "default_currency")
 
 	item_list = get_items(filters, additional_query_columns)
-	if item_list:
-		itemised_tax, tax_columns = get_tax_accounts(item_list, columns, company_currency)
```

Comment:
@harshmule-git  you removed this calling function from here but at line number 310 called function still exists